mxenabled/connect-widget
# Aw/ct 1060/light disclosure bug

#### Issue

https://mxcom.atlassian.net/browse/CT-1060 

#### Changes

- Removed experiment code and all experiments should now be handled by the host app calling the package

#### Testing Instructions

- Open connect and put your client on the HIDE_LIGHT_DISCLOSURE experiment and make sure that the light disclosure is not showing. 
- Take your client off the experiment and disable showMXBranding and make sure that the light disclosure is not showing
- Enable the showMXBranding flag and make sure that the light disclosure is showing
- No harm testing
